README.md

```diff
@@ -47,6 +47,7 @@ If you don't have a Metrics account already, [sign up](https://metrics.librato.c
 
 By default you can use `LIBRATO_USER` and `LIBRATO_TOKEN` to pass your account data to the middleware. While these are the only required variables, there are a few more optional environment variables you may find useful.
 
+* `LIBRATO_PROXY` - HTTP proxy to use when connecting to the Librato API (can also use the `http_proxy` environment variable commonly supported by Linux command line utilities)
```

Comment:
I think this should go between `LOG_LEVEL` and `EVENT_MODE`, these are ordered by likelihood of being needed so it definitely needs to go lower I think.